tests/phpunit/tests/link/getEditCommentLink.php

```diff
@@ -0,0 +1,102 @@
+<?php
+/**
+ * @group link
+ * @group comment
+ * @covers ::get_next_comments_link
+ */
+class Tests_Link_GetEditCommentLink extends WP_UnitTestCase {
+
+	public static $comment_ids;
+	public static $user_ids;
+
+	public static function wpSetUpBeforeClass( WP_UnitTest_Factory $factory ) {
+		self::$comment_ids = array(
+			'valid'   => $factory->comment->create( array( 'comment_content' => 'Test comment' ) ),
+			'invalid' => 12345, // Invalid comment ID
```

Comment:
No need to have a class member property specifically for an invalid ID. I think this should be simplified to only have `self::$comment_id` for the single _valid_ comment ID.